src/blocks/Investors.js

```diff
@@ -0,0 +1,124 @@
+import PreTitle from 'components/PreTitle'
+import BaseGrid from 'components/Grid'
+import React from 'react'
+import styled from 'styled-components'
+import get from 'utils/get'
+import Typography from 'components/Typography'
```

Comment:
```suggestion
import React from 'react'
import styled from 'styled-components'
import PreTitle from 'components/PreTitle'
import BaseGrid from 'components/Grid'
import Typography from 'components/Typography'
import get from 'utils/get'
```

Changing order of import to follow best practice + consistency: 
- external libraries first
- keep imports with same context together